pscript/stdlib.py

```diff
@@ -458,6 +485,45 @@ def get_all_std_names():
     } return a * b;
 }"""
 
+FUNCTIONS['op_set_union'] = """function (a, b) { // nargs: 2
+    let _union = new Set(a)
+    for (let elem of b) {
+        if (!_union.has(elem) && !FUNCTION_PREFIXop_contains(elem, _union)) { _union.add(elem) };
```

Comment:
Flake: too long lines.
```suggestion
        if (!_union.has(elem) && !FUNCTION_PREFIXop_contains(elem, _union)) {
            _union.add(elem)
        };
```